config/webpack.dev.js

```diff
@@ -0,0 +1,15 @@
+const { merge } = require('webpack-merge')
+const common = require('../webpack.common.js')
+
+module.exports = merge(common, {
+  devtool: 'inline-source-map',
+  mode: 'development',
+  devServer: {
+    contentBase: './dist',
```

Comment:
I am not sure, but I think here contentBase should point to the src directory.

Reply:
it should, dist is a 'build' folder that is being used only on production.